packages/styled-components/mixin.core.js

```diff
@@ -2,6 +2,15 @@ const { Mixin } = require('hops-mixin');
 
 class StyledComponentsMixin extends Mixin {
   configureBuild(webpackConfig, { jsLoaderConfig }, target) {
+    const { experimentalEsbuild } = this.options;
+
+    if (experimentalEsbuild) {
+      console.warn(
+        'The experimental esbuild transpilation does not support styled components yet!'
```

Comment:
It doesn't support it for SSR, right? But it would still give useful output in dev mode?
Maybe change this warning a bit, to indicate that it can still be used in dev mode.

Reply:
I haven't really looked into it so I'm not sure yet what works and what doesn't.